femitj/iReporter
# #162495503 Set up database

**What does this PR do?**
Sets up a Postgres database

**Description of the task to be completed**

Ensure the following tables are created
users table

**How should this be tested manually**
* clone the repo,
* cd into it and run npm install to install node modules. 
* Make sure you have nodejs and express installed. 
* Afterwards, `run npm start` to initiate the app and tables will be created.

**What are the relevant pivotal stories**
[#162495462](https://www.pivotaltracker.com/story/show/162495462)
